DEV-README.md

```diff
@@ -1,85 +1,169 @@
 # sos4R developer documentation
 
-## Package structure
+This file contains information for developers of the [R](http://r-project.org/) package `sos4R`.
+Documentation for usres can be found in the package's vignette (see `browseVignettes("sos4R")`).
 
-sos4R follows the regular R extension package structure. General documentation about R package development can be found at the following two websites.
+<!-- part of this page previously published at https://wiki.52north.org/Geostatistics/Sos4R -->
 
-* R extension packages by Hadley: http://r-pkgs.had.co.nz/ (well written with tight coupling to devtools and RStudio)
-* R extension manual: http://cran.r-project.org/doc/manuals/R-exts.html (generic, extensive)
+## Contribute
 
-## /R
+Please get in touch with the [community contact](https://52north.org/research/rd-communities/geostatistics/)
+of the geostatistics community and read the [Get Involved page](https://52north.org/software/get-involved/) if you want to *become a contributor* -- contributions are welcome!
 
-The actual source files in the ``/R`` directory follow a naming schema:
+You can also dive right in and joint the chat room: [![Join the chat at https://gitter.im/52North/sos4R](https://badges.gitter.im/52North/sos4R.svg)](https://gitter.im/52North/sos4R)
```

Comment:
Are you willing and is it possible for you to be always/often available in this gitter chatroom?
```suggestion
You can also dive right in and join the chat room: [![Join the chat at https://gitter.im/52North/sos4R](https://badges.gitter.im/52North/sos4R.svg)](https://gitter.im/52North/sos4R)
```